js/qz-tray.js

```diff
@@ -900,8 +909,8 @@ var qz = (function() {
                 if (data[i].constructor === Object) {
                     if ((!data[i].format && data[i].type && data[i].type.toUpperCase() !== 'RAW') //unspecified format and not raw -> assume file
                         || (data[i].format && (data[i].format.toUpperCase() === 'FILE'
-                        || (data[i].format.toUpperCase() === 'IMAGE' && !(data[i].data.indexOf("data:image/") === 0 && data[i].data.indexOf(";base64,") !== 0))
-                        || data[i].format.toUpperCase() === 'XML'))) {
+                            || (data[i].format.toUpperCase() === 'IMAGE' && !(data[i].data.indexOf("data:image/") === 0 && data[i].data.indexOf(";base64,") !== 0))
```

Comment:
This formatting looks inconsistent now...

Reply:
IntelliJ insisted on it, the indented block is in a sub block of the already subbed block above it which I'm sure is why it happened (it seems every minor update to the IDE comes with alterations to the default code style and it's a bit tiring).
Either you or I can play around with the code-style settings to see if it's possible to change it back.

Reply:
Sorry, I should have been more specific... the line above it is the problem.  Not the ones you touched.

Reply:
```js
if (foo || // spillover
      || bar
      || bar) {
   // do something
```

I've never seen it spill twice like that.  Are you sure that's the IDE's doing?  If so, fine.  Just strange. :)

Reply:
Oh, I see it now.  It's because of the grouping `( )`.  No worries then.